src/main/resources/application.yaml

```diff
@@ -84,10 +84,11 @@ quarkus:
           policy: authenticated
 
     limits:
-      # Let's limit low-level maximum HTTP request size to 5 megs: stricter limit (1 meg)
-      # is applied at the JSON API level. Low-level limits may result in EPIPE/413 errors
+      # Let's limit low-level maximum HTTP request size to 20 megs: stricter
+      # limit (4 meg per document) is applied at the JSON API level.
+      # Low-level limits may result in EPIPE/413 errors
       # whereas at higher level we can use regular JSON API error responses
-      max-body-size: 5M
+      max-body-size: 20M
```

Comment:
Can you add this to the CONFIGURATION.md file?

Reply:
Do we need to check limit on each document in insertMany? What if user add one document of 10 MB size with no index set. We will try to store the data into single doc_json field. Is that the intended change?